pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -54,10 +55,20 @@ python3Packages.buildPythonApplication rec {
       python-magic progressbar33 pypdf2 tlsh
     ])
     ++ lib.optionals stdenv.isLinux [ python3Packages.pyxattr python3Packages.rpm acl cdrkit dtc ]
-    ++ lib.optionals enableBloat ([
-      abootimg apksigcopier apksigner apktool cbfstool colord enjarify ffmpeg fpc ghc ghostscriptX giflib gnupg gnumeric
-      hdf5 imagemagick libcaca llvm jdk mono ocaml odt2txt oggvideotools openssh pdftk poppler_utils procyon qemu R tcpdump ubootTools wabt radare2 xmlbeans
-    ] ++ (with python3Packages; [ androguard binwalk guestfs h5py pdfminer-six ]));
+    ++ lib.optionals enableBloat (
+      [
+        apksigcopier apksigner enjarify ffmpeg fpc ghc ghostscriptX giflib gnupg pdftk
+        hdf5 imagemagick libcaca llvm jdk mono ocaml odt2txt openssh
+        poppler_utils procyon qemu R tcpdump wabt radare2 xmlbeans
+      ]
+      ++ (with python3Packages; [ androguard binwalk h5py pdfminer-six ])
+      # oggvideotools is broken on Darwin, please put it back when it will be fixed?
+      ++ lib.optionals stdenv.isLinux [ abootimg cbfstool colord ubootTools python3Packages.guestfs oggvideotools ]
```

Comment:
It would be nice to filter based on `lib.meta.availableOn`, to make this more accurate / maintainable.

(But please don't filter on meta.broken, as that's more likely to flip back and forth over time and surprise people.)

Reply:
Would you advise to filter the whole list or filter only parts of the list? Would you make this part of this PR or can we revisit this in another PR?

Reply:
I think it should be part of this PR, because you're introducing the `isLinux` condition in this PR, right?

Reply:
Correct

Reply:
> It would be nice to filter based on `lib.meta.availableOn`, to make this more accurate / maintainable.

That sounds like a really good idea to me.

Reply:
Can we also filter unfree packages like that?

Reply:
> Can we also filter unfree packages like that?

I don't think it's a good idea to do it, see one of the resolved discussion on that with @davidak

Reply:
I implemented the proposal, let me know if you see any other way to do it.

Reply:
> I don't think it's a good idea to do it, see one of the resolved discussion on that with @davidak

We don't need to make it dependend on config.allowUnfree